src/stroke2fill.js

```diff
@@ -0,0 +1,37 @@
+const Svg2 = require('oslllo-svg2');
+const Potrace = require('oslllo-potrace');
+
+function svgFromPathData(
+  pathData,
+  origWidth,
+  origHeight,
+  strokeWidth,
+  hasFillAttr
+) {
+  const fill = hasFillAttr ? 'fill="#000000"' : '';
+  const width = 600;
+  const height = (width / origWidth) * origHeight;
+  return (
+    '<?xml version="1.0" encoding="UTF-8"?>\n' +
+    `<svg width="${width}" height="${height}" viewBox="0 0 ${origWidth} ${origHeight}" ` +
+    `fill="none" xmlns="http://www.w3.org/2000/svg">\n` +
+    `  <path d="${pathData}" stroke="#000000" stroke-width="${strokeWidth}" ${fill}/>\n` +
+    '</svg>\n'
+  );
+}
+
+async function stroke2fill(pathData, width, height, strokeWidth, hasFillAttr) {
+  const svgBuffer = svgFromPathData(
+    pathData,
+    width,
+    height,
+    strokeWidth,
+    hasFillAttr
+  );
+  var pngBuffer = await Svg2(svgBuffer).png({ transparent: false }).toBuffer();
```

Comment:
Converting the SVG to a PNG and then to an SVG again seems like a huge process to only suppot stroke regarding the fact that not everyone need it. Indeed most of iconfont designers knows the limitations of iconfonts and then avoir using unsupported SVG features or at least convert the SVGs upstream. I'm not sure it worth it.

Reply:
Hello, in my case it has happened to me many times, the designers give you the icon and you have to find how to adapt it, in this case it seemed like a good addition to avoid wasting time. Even if it is a partial support.